cluster/juju/layers/kubeapi-load-balancer/reactive/load_balancer.py

```diff
@@ -35,6 +36,25 @@
 from subprocess import CalledProcessError
 
 
+apilb_nginx = """/var/log/nginx.*.log {
```

Comment:
question:

Why declare this inline instead of using the template system that exists and is used elsewhere? I'm not generally +/- for this approach, just curious why you chose the approach you did.

Reply:
I'm not +/- either way and mainly just selected this method randomly. I'd be happy to convert it over, but I felt that it would be more obvious to someone reading the code what was happening if they could easily find that data. It probably isn't worth it and should be a template though.

Reply:
Good enough to satisfy my curiosity. Thanks for the explanation